core/src/main/scala/kafka/server/KafkaRequestHandler.scala

```diff
@@ -335,6 +356,8 @@ class BrokerTopicStats extends Logging {
       topicMetrics.closeMetric(BrokerTopicStats.FailedProduceRequestsPerSec)
       topicMetrics.closeMetric(BrokerTopicStats.TotalProduceRequestsPerSec)
       topicMetrics.closeMetric(BrokerTopicStats.ProduceMessageConversionsPerSec)
+      topicMetrics.closeMetric(BrokerTopicStats.RemoteBytesOutPerSec)
```

Comment:
It seems all of the 5 new metrics are only applicable to the leader. If that's the case, should we remove all 5 of them here?

Reply:
The rest of the 3 metrics are used in followers too.

`RemoteBytesInPerSec` is emitted when bytes are copied from the remote tier to the broker. This can happen when a consumer fetches records from a broker, and the broker gets it from remote tier and responds. From Kafka 2.4.0, Fetch requests can be made to any broker part of the ISR, which can lead to `RemoteBytesInPerSec` to change. This is the same reason `BytesOutPerSec` is not in this method.

Same goes for `RemoteReadRequestsPerSec` and `FailedRemoteReadRequestsPerSec`. These can happen when a consumer fetches from any broker, and that fetches from the remote tier. This is common to leaders and followers.

Reply:
That makes sense, thanks!